docs/release/release_0_3_0.md

```diff
@@ -113,14 +115,14 @@ case, we provide an equivalent API for the same functionality.
   to any layer type.
 - `add_image(path=...)` and `add_labels(path=...)` have been removed. Users
   should use `viewer.open(...)` instead.
-- image pyramids are no longer automatically generated when a dataset is large.
+- Image pyramids are no longer automatically generated when a dataset is large.
   This should not affect API compatibility but might affect performance. For
   most users, this should result in faster startup for large images.
 - `add_image(..., is_pyramid=False)` is now `add_image(..., multiscale=False)`.
   This will allow us to use a consistent keyword argument when we add
   multiscale support for other layer types.
 - `layer.to_svg()` has been removed. This functionality is now implemented with
-  `viewer.save('path/to/layer.svg', layer)`.
+  `viewer.save('path/to/layer.svg', layer, plugin='svg')` through our plugin architecture.
```

Comment:
I'm ok with this comment, but shouldn't the plugin be automatically selected given that extension?

Reply:
>  shouldn't the plugin be automatically selected given that extension

You'll get the first plugin in order that can handle an `svg` extension which depending on what they have installed might or might not be this one. Probably better to be explicit in this context.